agent/exec/container/executor.go

```diff
@@ -41,12 +41,35 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 		}
 	}
 
+	// add v1 plugins to 'plugins'
 	addPlugins("Volume", info.Plugins.Volume)
```

Comment:
Right, but now there is a field called `Plugins` in docker info, that is essentially deprecated without being deprecated. Anyway, not sure what's the best solution.